docs/source/migration-guides/0.11-serialization.md

```diff
@@ -0,0 +1,100 @@
+# Adjusting code to changes in serialization API introduced in 0.11
+
+## Background
+
+When executing a statement through the CQL protocol, values for the bind markers are sent in a serialized, untyped form. In order to implement a safer and more robust interface, drivers can the information returned after preparing a statement to check the type of the data provided by the user against the actual types of the bind markers.
```

Comment:
`drivers can the information` - did you mean `can use`?

Reply:
Yes - thanks, corrected.